src/main/kotlin/com/zighang/core/application/ObjectStorageService.kt

```
@@ -0,0 +1,92 @@
+    ) : String {
+        var metadata: ObjectMetadata = ObjectMetadata()
+        metadata.contentLength = file.size
+        metadata.contentType = file.contentType
+
+        try {
+            var putRequest: PutObjectRequest? = PutObjectRequest(
+                bucketName, fileName, file.inputStream, metadata
+            ).withCannedAcl(CannedAccessControlList.PublicRead)
+
+            amazonS3Client.putObject(putRequest)
+        } catch(e: IOException){
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)